packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -78,6 +85,12 @@ import type { Nullable, Optional, StreamRoles } from '@speckle/shared'
 import { useDebouncedTextInput, type InfiniteLoaderState } from '@speckle/ui-components'
 import { MagnifyingGlassIcon, Squares2X2Icon } from '@heroicons/vue/24/outline'
 
+graphql(`
+  fragment ProjectsHiddenProjectWarning_UserProjectCollection on UserProjectCollection {
```

Comment:
I'd still like to see the parent component name in there. Maybe `ProjectsDashboard_HiddenProjectWarning_UserProjectCollection`?

Or just `ProjectsDashboard_UserProjectCollection` cause that's how we usually approach naming these. Even though this component has its own query, if there's fragment requirements that it has outside of the ones defined by child components, it makes sense defining those in its own fragment, I think.